examples/pi_series_omega_control.py

```diff
@@ -1,36 +1,52 @@
+
 sys.path.append(os.path.abspath(os.path.join('..')))
 import gym_electric_motor as gem
 from gym_electric_motor import reference_generators as rg
 from gym_electric_motor.visualization import MotorDashboard
 
+"""Run this file from within the 'examples' folder:
+>> cd examples
+>> python pi_series_omega_control.py
+Description:
+        Environment to control a Continuously controlled DC Series Motor.
+
+        Controlled Quantity: 'omega'
+
+        Limitations: Physical limitation of the  motor Current
+
+        Converter : OneQuadrantConverter from converters.py
+"""
 
 if __name__ == '__main__':
-    env = gem.make(
-        'DcSeriesCont-v1',   # replace with 'DcSeriesDisc-v1' for discrete controllers
-        # Pass an instance
-        #visualization=MotorDashboard(plotted_variables='all', visu_period=1),
-        visualization=MotorDashboard(plots=['omega','reward', 'i'], dark_mode=True),
-        motor_parameter=dict(r_a=15e-3, r_e=15e-3, l_a=1e-3, l_e=1e-3),
-        # Take standard class and pass parameters (Load)
-        load_parameter=dict(a=0.01, b=.1, c=0.1, j_load=.06),
-
-        # Pass a string (with extra parameters)
-        ode_solver='scipy.solve_ivp', solver_kwargs=dict(),
-        # Pass a Class with extra parameters
-        reference_generator=rg.SwitchedReferenceGenerator(
-            sub_generators=[
-                rg.SinusoidalReferenceGenerator, rg.WienerProcessReferenceGenerator(), rg.StepReferenceGenerator()
-            ], p=[0.1, 0.8, 0.1], super_episode_length=(1000, 10000)
-        )
-    )
-    controller = Controller.make('pi_controller', env)  # works for 'on_off'(disc),three_point(disc),'p_controller', 'cascaded pi' too
+    """
+           Continuous mode: The action is the average (normalized) voltage per time step which is assumed to be transferred to a PWM 
+                            converter to generate the switching sequence.
+
+           Discrete mode: The action is the switching state of the power converter i.e., a quantity from a discrete set of
+                          switching states which can be generated by the converter.   
+
+
+    """
+
+    env = gem.make('DcSeriesCont-v1',  # replace with 'DcSeriesDisc-v1' for discrete controllers
+                   state_filter=['omega', 'i'],
+                   # Pass an instance
+                   visualization=MotorDashboard(plots=['i', 'omega']),
+                   # Take standard class and pass parameters (Load)
+                   motor_parameter=dict(r_a=15e-3, r_e=15e-3, l_a=1e-3, l_e=1e-3),
+                   load_parameter=dict(a=0, b=.1, c=.1, j_load=0.04),
+                   # Pass a string (with extra parameters)
+                   ode_solver='scipy.solve_ivp', solver_kwargs={},
+                   # Pass a Class with extra parameters
+                   reference_generator=rg.WienerProcessReferenceGenerator())
+
+    controller = Controller.make('pi_controller',env)
```

Comment:
When testing the code the PI controller resulted in frequency overcurrent situations and, therefore, leading to episode terminations. Please either change the above motor parameter in such a way that even a single PI speed controller runs with only ocassional overcurrents or rearrange the example to use a cascaded PI controller for speed and current.